### DISCUSSION TITLE
Splash.dart issue

### DISCUSSION THREAD

**Discussion by ghost** [Imperial-lord/mapbox-flutter#13] [Q&A]
Hi Satya,
I am facing an issue with splash.dart.

Issue description,
package:mapbox_turn_by_turn/ui/splash.dart:47:10
Err Message,
<asynchronous suspension>
E/flutter ( 1818): #1      _SplashState.initializeLocationAndSave (package:mapbox_turn_by_turn/ui/splash.dart:47:10)
E/flutter ( 1818): <asynchronous suspension>

Please help...

**Comment by Imperial-lord** [comment]
```    
// Get the current user address
    String currentAddress =
        (await getParsedReverseGeocoding(currentLocation))['place'];
```

Looks like this line - Did you give the permissions for location? Can you check the output of the following line before assignment into the String variable?
